spec/ParseUser.spec.js

```diff
@@ -1428,6 +1466,47 @@ describe('Parse.User testing', () => {
     });
   });
 
+  fit_exclude_dbs(['postgres'])("link multiple providers and updates token", (done) => {
```

Comment:
Probably want to change from fit to it here as I think this will force a single test